### PR TITLE
feat(anmn_metadata): config content.ftl of mooring_hourly_timeseries_map

### PR DESCRIPTION
**Pull request by ocehugo**
Provide a basic content.ftl for the mooring hourly timeseries product

### CODE REVIEW

File: workspaces/imos/JNDI_anmn_metadata/moorings_hourly_timeseries_map/content.ftl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<h3>Moorings - Hourly time-series product</h3>
+
+<#list features as feature>
+  <div class="feature">
+    <h4>${feature.site_code.value}</h4>
+    <b>LON/LAT</b> ${feature.longitude.value}/${feature.latitude.value}<br/>
+    <b>Deployment start </b>${feature.time_coverage_start.value}<br/>
+    <b>Deployement end  </b>${feature.time_coverage_end.value}<br/>
+    <b>Date created </b> ${feature.date_created.value}<br/>
+    <b>Variables: </b> ${feature.standard_names.value}<br/>
```
Comment (mhidas) [inline workspaces/imos/JNDI_anmn_metadata/moorings_hourly_timeseries_map/content.ftl]: Let's be explicit here too, and call this "CF standard names"

Comment (ocehugo) [reply workspaces/imos/JNDI_anmn_metadata/moorings_hourly_timeseries_map/content.ftl]: IMO, at that stage/view they just want to know which variables are in and not be bothered to know what "CF standard names" means.

**Comment by ocehugo** [comment]
done - see opinion above

File: workspaces/imos/JNDI_anmn_metadata/moorings_hourly_timeseries_map/content.ftl (hunk 1 above)
Comment (mhidas) [reply workspaces/imos/JNDI_anmn_metadata/moorings_hourly_timeseries_map/content.ftl]: Ok, you might be right.

**Pull request merged**